Tasks/AzurePowerShellV4/InitializeAz.ps1

```diff
@@ -69,6 +95,16 @@ if ($endpointObject.scheme -eq 'ServicePrincipal') {
         throw (New-Object System.Exception("There was an error with the service principal used for the deployment.", $_.Exception))
        
     }
+
+    if($scopeLevel -eq "Subscription")
+    {
+        $SubscriptionId = $endpointObject.subscriptionId
+        $TenantId = $endpointObject.tenantId
+        $additional = @{ TenantId = $TenantId }
```

Comment:
We have one argument with splatting and other is without splatting. Is it intentional ?

Reply:
It is handled in the same way as used in other places.